### PR TITLE
Added Buttons

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Added 

- Primary , Secondary Buttons
- Icon Buttons
- Link Buttons
- Floating Buttons

### CODE REVIEW

File: components/buttons/buttons.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    background-color: var(--like-btn);
+}
+.apex-share-btn{
+    background-color: var(--share-btn);
+}
+.apex-subscribe-btn{
+    background-color: var(--subscribe-btn);
+}
+.floating-btn{
+    color: white;
```
Comment (Priya-Patel31) [inline components/buttons/buttons.css]: style: don't use colors directly make use of variables .

File: components/buttons/buttons.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    font-size: 20px;
```
Comment (Priya-Patel31) [inline components/buttons/buttons.css]: fix : change font-size in rems

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge

File: components/buttons/buttons.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    color: var(--apex-text-link-color);
+    text-decoration: none;
+}
+.apex-link-btn{
+    background-color: var(--link);
+}
+.apex-cart-btn{
+    background-color: var(--cart-btn);
+}
+.apex-like-btn{
```
Comment (Priya-Patel31) [inline components/buttons/buttons.css:25]: Just give some margin right between icon and text inside button.

**Review by Priya-Patel31 — APPROVED** [review]
Just you have some minor changes rest everything seems perfect in UI and in code too.

File: components/buttons/buttons.css (hunk 3 above)
Comment (atharvabhanage02) [reply components/buttons/buttons.css:25]: Yes I 'll do that ,that would make it look good

**Pull request merged**